apps/jobs/views.py

```diff
@@ -2073,3 +2074,39 @@ def get_github_badge_data(
         else:
             data["message"] = f"{challenge_obj.title}"
     return Response(data, status=http_status_code)
+
+
+@api_view(["GET"])
+@throttle_classes([UserRateThrottle])
+@permission_classes((permissions.IsAuthenticated, HasVerifiedEmail))
+@authentication_classes((ExpiringTokenAuthentication,))
+def challenge_phase_submission_count_by_status(request, challenge_phase_id):
```

Comment:
Let's stick to one convention -- `challenge_phase_id` -> `challenge_phase_pk`. I know there are discrepancies in the codebase but we can fix them up in pieces.